### DISCUSSION TITLE
Can we name the telescope logo/mascot Ollie?

### DISCUSSION THREAD

**Discussion by paigerduty** [open-telemetry/community#1269] [Community Discussions]
Pretty please? 

cc @austinlparker

**Comment by bhs** [comment]
"Telly"?

**Comment by alolita** [comment]
How about O11y? O+11characters+y = Observability & supporting the Star wars naming convention 😁 e.g. r2d2

**Reply by cartermp** [reply]
I like O11y, but because that's also kind of a real acronym it might be kinda weird. But I could see this being fine.

**Reply by puckpuck** [reply]
I like Ollie.

**Reply by svrnm** [reply]
A few variations:

o11yver
o11yot 
o11ymon
o11yscope

**Reply by kenfinnigan** [reply]
What about o11ie? If we want to get away from the real acronym

**Reply by paigerduty** [reply]
o11ie is cute!

**Comment by melbgirl** [comment]
Can we get a mascot figure designed to match? The telescope image (as is) is an excellent logo, but not a mascot. There needs to be some ‘life’ in the image, and probably some anthropomorphism too.

**Reply by cartermp** [reply]
Probably not gonna happen, but it would be probably more unique to have a telescope figure in the style of Courage the Cowardly Dog.

**Comment by austinlparker** [comment]
Maybe when we graduate we can get a Phippy.
